lib/src/widgets/code_block_component_widget.dart

```diff
@@ -146,15 +149,15 @@ class _CodeBlockComponentWidgetState extends State<CodeBlockComponentWidget>
               ? TextSpan(text: node.value)
               : TextSpan(
                   text: node.value,
-                  style: builtInCodeBlockTheme[node.className!],
+                  style: widget.theme[node.className!],
```

Comment:
Watch out. The null assertion operator `!` doesn't give a stack trace when it throws an error. You might wan to check that the `node.className` is not null (if it is then throw an exception rather than using `!`).